manifests/pip.pp

```diff
@@ -135,14 +135,21 @@
     fail('python::pip cannot provide uninstall_args with ensure => present')
   }
 
+  $real_pkgname = $pkgname
+  if $pkgname =~ /==/ {
+    $parts = split($pkgname, '==')
+    $real_pkgname = $parts[0]
```

Comment:
you reassign `$real_pkgname`. This should not work. The assignment in line 128 should probably happen in an else block. Can you also add a test where the pkgname contains a `==`?

Reply:
Made this fix, and added the test.